### ISSUE TITLE
Special product menu idea

### ISSUE THREAD

**Issue by Vartkat** [zcadditions/ZCA-Bootstrap-Template-for-1.5.6-v2.0.0c#149]
Hi,

First thank you very much for this template, I agree with someone on ZC forum, it should be included as it is the easier I found to customize.
I'm using it with an old Treemenu from 1.3.8, which still works with my 1.5.5e and disappears when it's on mobile width.
I know this sounds a lot, but I think it could make this template even better so I submit the idea.
When the template is in mobile config, i.e. menu hidden under the hamburger menu, I feel it's not so obvious this is the menu where to find the products, more because (in my case the hamburger is staying up the page.

In fact this is two ideas 
- making a second "hamburger menu" containing only the products
- making this menu icon floating always at the left side of the screen, 
- Making this menu a drawer coming from the left side of the screen.

This way, wherever the customer is down the page he ca always pull the product drawer to navigate.
With only one hambuger menu, customer has to
- go up the page, 
- pull the hamburger
- pull the product submenu
- click for a category

With my "product hamburger" he can
- pull the product menu (as it's always there)
- ask a category

Hope to see it soon, I think this is not so much work as I feel (I'm not really a coder) it's a reduced version of the off_canvas one done as a sidebox that could be (or not) activated.

Thanks for reading

V.

**Comment by Vartkat** [comment]
OK. Let me rephrase...

As the Dynamic Categories tree menu (https://www.zen-cart.com/downloads.php?do=file&id=396) already works with ZCA-Bootstrap template, I think what I need is just a few steps away.

With ZCA-Bootstrap, when viewed on mobile phone, the left columns disappears. That's fine, but what I would like is a little icon to show that even if the column is not visible it's still there. The best would be that the icon stays static whatever is the scrolling in the page.

If the reader clicks on the icon, the left column comes back like a drawer from the left side of the screen.
You can see an example at https://www.codeply.com/go/SufxS3ir55.

V.

**Comment by zcadditions** [comment]
I choose not to include yet another menu but instead included a "back to top" button and a fixed header that scrolls with page.
